packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.h

```diff
@@ -103,6 +103,24 @@ extern FWFWKNavigationActionData *FWFWKNavigationActionDataFromNavigationAction(
  */
 extern FWFNSUrlRequestData *FWFNSUrlRequestDataFromNSURLRequest(NSURLRequest *request);
 
+/**
+ * Converts a WKNavigationResponse to an FWFWKNavigationResponseData.
+ *
+ * @param response The object containing information to create a WKNavigationResponseData.
+ *
+ * @return A FWFWKNavigationResponseData.
+ */
+extern FWFWKNavigationResponseData *FWFWKNavigationResponseDataFromNavigationResponse(
```

Comment:
nit: We recently decided on a new naming scheme for these to include the word `Native` to make it easier to read. Can you change this to `FWFWKNavigationResponseDataFromNativeNavigationResponse`

```suggestion
extern FWFWKNavigationResponseData *FWFWKNavigationResponseDataFromNativeNavigationResponse(
```